httpx/__init__.py

```diff
@@ -43,6 +43,21 @@
 from ._transports.wsgi import WSGITransport
 from ._types import AsyncByteStream, SyncByteStream
 
+try:
+    from ._main import main
+except ImportError:  # pragma: nocover
+
+    def main() -> None:  # type: ignore
+        import sys
+
+        print(
+            "The httpx command line client could not run because the required "
+            "dependancies were not installed.\nMake sure you've installed "
```

Comment:
Think this is a typo 🙂 
```suggestion
            "dependencies were not installed.\nMake sure you've installed "
```